hyperspy/_signals/eds_tem.py

```diff
@@ -653,3 +653,52 @@ def _get_dose(self, method, beam_current='auto', real_time='auto',
             return real_time * beam_current * 1e-9 / constants.e
         else:
             raise Exception('Method need to be \'zeta\' or \'cross_section\'.')
+
+    def determine_zeta_factor(self, intensities, composition, thickness, density):
+        """
+        Determine the zeta-factors from a known sample.
+
+        Parameters
+        ----------
+        intensities: list of signal
+            The intensity for each X-ray line.
+        composition: list of float or list of signal
```

Comment:
Is there anyway to make to composition default to one all over. Assuming most people would use pure standard samples as was written in the original paper?